flac.md

```diff
@@ -237,7 +237,7 @@ NOTES
   - 14 - During recording
   - 15 - During performance
   - 16 - Movie/video screen capture
-  - 17 - A bright coloured fish
+  - 17 - A bright colored fish
```

Comment:
Sorry to be extremely pedantic, but the British spelling 'coloured' seems to be the spelling used not just here, but also in ID3 and most likely is not a true typo. Are we standardizing to American English  spellings?